test_projects/ios/EarlGreyExample/ops.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+EARL_GREY_EXAMPLE="$TEST_PROJECTS_IOS/EarlGreyExample"
+
+function setup_ios_env() {
+  if ! [ -x "$(command -v xcpretty)" ]; then
+    gem install cocoapods
+  fi
+  (cd "$EARL_GREY_EXAMPLE" && pod install)
+}
+
+function install_xcpretty() {
+  if ! [ -x "$(command -v xcpretty)" ]; then
+    gem install xcpretty
```

Comment:
You can specify xcpretty version too, but it's just used for formatting build output so it won't break anything.

Reply:
I will leave it as is